src/System.Text.Json/tests/JsonElementWriteTests.cs

```diff
@@ -460,6 +464,122 @@ public static void WriteNumberScientificAsProperty(bool indented)
                 "{\"m\\u00eal\\u00e9e\":1e6}");
         }
 
+        [Fact]
+        public static void WriteValueUnicodeEscapeSequence()
+        {
+            string unicodeString = "\u0421\u0430\u0439\u043D \u0443\u0443? \u04E9\u04E9";
+            string json = $"[\"{unicodeString}\"]";
+            var buffer = new ArrayBufferWriter<byte>(1024);
+            ReadOnlySequence<byte> sequence = JsonTestHelper.GetSequence(Encoding.UTF8.GetBytes(json), 1);
+            string expectedStr = GetEscapedExpectedString(unicodeString, StringEscapeHandling.EscapeNonAscii);
+
+            using (JsonDocument doc = JsonDocument.Parse(sequence, s_options))
+            {
+                JsonElement target = doc.RootElement[0];
+
+                using (var writer = new Utf8JsonWriter(buffer))
+                {
+                    target.WriteTo(writer);
+                    writer.Flush();
+                }
+
+                AssertContents(expectedStr, buffer);
+            }
+        }
+
+        [Fact]
+        public static void WriteValueSurrogatesEscapeSequence()
+        {
+            string unicodeString = "\uD800\uDC00 \uD803\uDE6D\uD834\uDD1E \uDBFF\uDFFF";
+            string json = $"[\"{unicodeString}\"]";
+            var buffer = new ArrayBufferWriter<byte>(1024);
+            byte[] temp = Encoding.UTF8.GetBytes(json);
+            ReadOnlySequence<byte> sequence = JsonTestHelper.GetSequence(temp, 1);
+            string expectedStr = GetEscapedExpectedString(unicodeString, StringEscapeHandling.EscapeNonAscii);
+
+            using (JsonDocument doc = JsonDocument.Parse(sequence, s_options))
+            {
+                JsonElement target = doc.RootElement[0];
+
+                var writer = new Utf8JsonWriter(buffer);
```

Comment:
It looks like only some of the new tests put writers are in using statements; is that intentional?

Reply:
I see everywhere were else it is used without using, following existing pattern

Reply:
Please add the using to all new Utf8JsonWriter instances.

Reply:
sure